### ISSUE TITLE
New DataPalooza EventBrite

### ISSUE THREAD

**Issue by jhibbets** [ncopenpass/NCOpenPass#1276]
Hi all, I created a new EventBrite with the DataPalooza ticket types. We already had one person register, so I manually added them.

https://www.eventbrite.com/e/nc-open-pass-datapalooza-tickets-50593487483

A few things:
- [x] Can I get a second set of eyes on it just to review and make sure all the details are correct
- [x] Do we need to add a questions about dietary restrictions?
- [x] Do we need to added a question about T-shirt size

**Comment by jhibbets** [comment]
Note: I also updated our blog post registration link. If reg for DataPalooza exists somewhere else, we need to update that as well.

**Comment by ChrisTheDBA** [comment]
It looks fine. I didn't see anything.

Yes, we need questions on dietary and t-shirts if we plan on feeding them
or giving them t-shirts.

We also need to move over all the people that bought Open Pass tickets and
the sponsor and other discount codes.

I'm also curious why the ticket fees went from making the person pay to us
absorbing the fee. I'm ok with absorbing it, but you said you had a reason
for doing it the first way for Civic Camp and now are doing it differently
for Datapalooza. You never shared it and I'd like to understand better.

Chris Mathews

**Comment by jcwlib** [comment]
Just to clarify - we haven't finalized the exact times of the event.  When we did the walkthrough at Ipreo we had some small logistics questions to work out.  So the times might change.

**Comment by jcwlib** [comment]
I took the existing link out of the follow-up message we sent to the Civic
Camp registrants this morning.

That's the only place I know of that had the link to DataPalooza
registration.

**Comment by jhibbets** [comment]
I will send a notice to all open pass holders to register, I'm waiting on
the ATO code to get all that into one message. I'm planning on getting that
out this week.

re: fees, to be honest, it was a mistake. EB defaults to passing on fees
and when the Open Pass ticket was created way back in March, it was an
oversite. Since people had already bought the ticket, I thought it was best
to just leave it as-is and make all the other ticket types absorb the
costs. Make sense?

**Comment by jhibbets** [comment]
Questions about t-shirts and food have been added.

**Comment by jcwlib** [comment]
But the food question is required and only has Vegetarian options =) Need a
none in there as well

**Comment by ChrisTheDBA** [comment]
I tweaked it. Checked out at the same time.

Chris Mathews

**Comment by jhibbets** [comment]
Hrmm, that's strange, when I added the questions, I thought I left it as type Text (to leave it open ended)

**Comment by jhibbets** [comment]
Anyway, sounds like we are good here. Please open a new card for anything else.

**Comment by ChrisTheDBA** [comment]
if it's open ended, then we have 50 different answers for 50 registrations.

This is the list for Open Pass
[image: image.png]

Chris Mathews
